documentation/vespa7-release-notes.html

```diff
@@ -68,10 +112,12 @@ <h3 id="services-xml">services.xml</h3>
   </thead>
   <tbody>
     <tr>
+      <!-- TODO -->
```

Comment:
What does TODO mean here (and many other places)?

Reply:
When the item is not done/not known by me to be done. To make doubly sure we don't release Vespa 7 with any of these items not done.

Reply:
All the stuff I added has been done on 7, except for the document/v1 stuff that I commented out.

Reply:
The protocol is to only add stuff that has been done. Everything else should be on the Vespa 7 theme page.

Reply:
Thanks, removed the TODOs on metrics, rotationscheme and container.filter

Reply:
Yes, I see.